sdcm/tester.py

```diff
@@ -762,6 +757,33 @@ def clean_resources(self):
         if self.monitors is not None:
             self.monitors.get_backtraces()
             self.monitors.download_monitor_data()
+
+            # upload prometheus data
+            prometheus_folder = glob.glob(os.path.join(self.logdir, '*monitor*'))[0]
+            file_path = os.path.join(self.logdir, os.path.basename(os.path.normpath(self.job.logdir)))
```

Comment:
non very clean messages for me)
but okay, I create archive like job-2017-06-01T10.26-6ff133e.zip(with only prometheus folder) inside of self.job.logdir